src/handlers/motions/motionStaked/motionStaked.ts

```diff
@@ -48,11 +51,14 @@ export default async (event: ContractEvent): Promise<void> => {
       requiredStake,
     );
 
+    const updatedMessages = getUpdatedMessages(messages, requiredStake, motionStakes, transactionHash, logIndex, vote, staker, amount);
```

Comment:
There's quite a few props here. Maybe you'd be better off having a single argument that's an object, with the current arguments as object properties. Then you could do something like:

```js
getUpdatedMessages({
  messages, 
  requiredStake, 
  motionStakes, 
  messageKey: `${transactionHash}${logIndex}`
  ...args
})
```

It's a little better and means you don't have to worry about ordering.